### ISSUE TITLE
Obsidian fails to load plugin

### ISSUE THREAD

**Issue by sm18lr88** [bramses/summarize-with-gpt3-obsidian#3]
Windows 11, latest Obsidian (1.1.9): 

- Downloaded the repo and cloned it to my plugin folder as instructed.
- Opened Obsidian → Opened community plugins: GPT3 summarizer shows up but turned off → on switching the button on I get the error: "Failed to load plugin obsidian-gpt3-summarizer".

I haven't modified any of the files.
My obsidian vault is located in a separate NTFS partition, not drive C:\. I don't know if that affects anything.

Any clues? Do I need certain dependencies? 
Thanks.

**Comment by ChrisSwan** [comment]
I get the same error, and in the console it's looking for a main.js file:

Plugin failure: obsidian-gpt3-summarizer Error: ENOENT: no such file or directory, open 'C:\<mypath>\Vault\.obsidian\plugins\summarize-with-gpt3-obsidian-master\main.js'

**Comment by DeLub** [comment]
I had the same. Apparently the main.js is now main.ts??

If you install the BRAT plugin and add this repository there, it does work, though.

Hope that helps.

**Comment by ChrisSwan** [comment]
Thanks @DeLub using the beta version with BRAT fixed the issue for me.

**Comment by sm18lr88** [comment]
Yes, thanks @DeLub, it works with BRAT. Consider the issue "fixed", but perhaps this should be added to install instructions.
